libs/ui-kit/src/lib/sort-select.tsx

```diff
@@ -85,25 +85,15 @@ export function SortSelect({
         }}
       </MenuButton>
 
-      <Transition
-        as={Fragment}
-        enter="ease-out duration-100"
-        enterFrom="opacity-0 scale-95"
-        enterTo="opacity-100 scale-100"
-        leave="ease-in duration-75"
-        leaveFrom="opacity-100 scale-100"
-        leaveTo="opacity-0 scale-95"
+      <MenuItems
+        className={clsx(
+          'absolute left-[0px] z-10 mt-[4px] w-fit min-w-full origin-top rounded-[8px] bg-[#0D0D0E] py-[8px] shadow-lg focus:outline-none',
+          'max-h-[256px] overflow-y-scroll',
+          'border-haqq-border border-[1px]',
```

Comment:
just wanted to ask if there's any need in writing like that: `border-[1px]` , `left-[0px],` 
when you can write like that: `border` , `left-0`

<img width="296" alt="Screenshot 2024-08-26 at 12 28 52" src="https://github.com/user-attachments/assets/ca096d54-b25a-4f5c-8b89-e1c456ed9596">

<img width="421" alt="Screenshot 2024-08-26 at 12 28 04" src="https://github.com/user-attachments/assets/cf5f12ec-bbe8-45dd-b4c5-038a87cadcc6">

Reply:
actually it doesn't matter cuz it generate same rules in result css.
but this values dosen't rely on tailwind configuration.